js/screenSnippet/index.js

```diff
@@ -144,10 +144,14 @@ function readResult(outputFileName, resolve, reject, childProcessErr) {
         try {
             // convert binary data to base64 encoded string
             let output = Buffer.from(data).toString('base64');
-            resolve({
+            const resultOutput = (isWindowsOS) ? {
```

Comment:
This can be written as
```suggestion
           resolve({
                type: isWindowsOS ? 'image/png;base64' : 'image/jpg;base64',
                data: output
            });
```

Reply:
:+1: